extensions/kube-object-event-status/package.json

```diff
@@ -8,18 +8,15 @@
     "styles": []
   },
   "scripts": {
-    "build": "webpack && npm pack",
-    "dev": "webpack --watch",
+    "build": "npx webpack && npm pack",
+    "dev": "npx webpack -- --watch",
     "test": "echo NO TESTS"
   },
   "files": [
     "dist/**/*"
   ],
-  "dependencies": {},
   "devDependencies": {
     "@k8slens/extensions": "file:../../src/extensions/npm/extensions",
-    "ts-loader": "latest",
-    "typescript": "latest",
-    "webpack": "latest"
+    "npm": "^8.5.3"
```

Comment:
npx in npm6 is buggy, that's why bundled extensions now have npm@8

Reply:
Do we need `npx`?

Reply:
`npx` is the thing that allows us to remove webpack/typescript/ts-loader etc.